docs/core/tools/csproj.md

```diff
@@ -194,9 +194,13 @@ A human-friendly title of the package, typically used in UI displays as on nuget
 ### Authors
 A semicolon-separated list of packages authors, matching the profile names on nuget.org. These are displayed in the NuGet Gallery on nuget.org and are used to cross-reference packages by the same authors.
 
-### Description
+### PackageDescription
+
 A long description of the package for UI display.
 
+### Description
+A long description of the package for UI display. This property is used if `PackageDescription` is not specified. It is also used to specify the generated assembly's description.
```

Comment:
Thank you for opening this PR! 

After reading through your linked visual studio feedback and the original page, I think we can improve this even further. According to the [AssemblyInfo section](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#assemblyinfo-properties), the **Description** property is really the description of the assembly I think we should describe it that way first and then state that this property will be used as the package description of that **PackageDescription** property is not specified.

Reply:
I agree.

Reply:
Sorry for the delay. I updated the description. 

I noticed that the attributes defined on this page are very inconsistent in how they describe things. Examples:
Authors - A semicolon-separated list
Copyright - Copyright details
PackageOutputPath - Determines the
IncludeSymbols - This Boolean value
Istool  - Specifies whether

I used the terminology that PackageDescription was using but I think this page could stand to have the descriptions reviewed to make them read like they are not copied from documentation elsewhere.